Project/Source/Modules/ModuleProject.cpp

```diff
@@ -313,6 +317,19 @@ bool ModuleProject::Init() {
 	return true;
 }
 
+UpdateStatus ModuleProject::Update() {
+	for (ComponentScript& script : App->scene->scene->scriptComponents) {
+		if (App->time->HasGameStarted() && App->scene->sceneLoaded) {
+			Script* scriptInstance = script.GetScriptInstance();
+			if (scriptInstance != nullptr) {
+				scriptInstance->Update();
```

Comment:
Why here instead of the Component Update?

Reply:
Because if we add a GameObject or Component in the middle of an iteration it crashes.